client/platform/web-girder/api/viame.service.ts

```diff
@@ -25,6 +25,7 @@ export interface BrandData {
   logo: string;
   name: string;
   loginMessage: string;
+  alertMessage: string;
```

Comment:
Most of the time this value will be empty.

`alertMessage: string | null`  should be the type.

Reply:
Should this still be `string | null` or should it just be optional?